be/src/vec/olap/olap_data_convertor.cpp

```diff
@@ -138,6 +138,7 @@ void OlapBlockDataConvertor::OlapColumnDataConvertorBase::set_source_column(
         auto nullable_column =
                 assert_cast<const vectorized::ColumnNullable*>(_typed_column.column.get());
         _nullmap = nullable_column->get_null_map_data().data();
+        _nullmap += row_pos;
```

Comment:
`row_pos` has been already used to compute offset of _nullmap somewhere (HLL, BitMap, String, etc.). Maybe some basic types `row_pos` is missing to compute `_nullmap` and we should only fix this in those types.